arrow-core-data/build.gradle

```diff
@@ -11,6 +11,7 @@ apply from: "$DOC_CREATION"
 
 dependencies {
     api project(":arrow-annotations")
+    api project(":arrow-continuations")
```

Comment:
I don't think api visibility is a good idea for this as this isn't meant to be used by anyone using core directly.

Reply:
yeah, that is a good point! Will use `implementation` here 👍